cli/pcluster/utils.py

```diff
@@ -256,43 +256,6 @@ def _get_json_from_s3(region, file_name):
     return json.loads(file_contents)
 
 
-def get_supported_features(region, feature):
-    """
-    Get a json object containing the attributes supported by a feature, for example.
-
-    {
-        "Features": {
-            "efa": {
-                "instances": ["c5n.18xlarge", "p3dn.24xlarge", "i3en.24xlarge"],
-                "baseos": ["alinux", "centos7"],
-                "schedulers": ["sge", "slurm", "torque"]
-            },
-            "batch": {
-                "instances": ["r3.8xlarge", ..., "m5.4xlarge"]
-            }
-        }
-    }
-
-    :param region: AWS Region
-    :param feature: the feature to search for, i.e. "efa" "awsbatch"
-    :return: json object containing all the attributes supported by feature
-    """
-    try:
-        features = _get_json_from_s3(region, "features/feature_whitelist.json")
```

Comment:
It seems to me we can get rid of the `_get_json_from_s3` utility function.

Reply:
done